src/components/AppToolbar.tsx

```diff
@@ -289,6 +292,15 @@ class AppToolbarInternal extends React.Component<AppToolbarInternalProps> {
             <MdHelpOutline />
             <IconText>{t("Help")}</IconText>
           </ToolbarLink>
+
+          <Dropzone onDrop={this.props.onFileSelected}>
+            {({getRootProps, getInputProps}) => (
+              <div {...getRootProps({className: 'dropzone maputnik-toolbar-link'})}>
+                <input {...getInputProps()} />
+                Drop file here
```

Comment:
This needs translation...

Reply:
This came from https://react-dropzone.js.org/#section-class-components, or do you mean i18n?

Reply:
It not clear that this Dropzone relates only to a pmtiles file. As a user I'd expect that I could drag and drop a style json. 
If feasable, how about a full screen drop zone, where the user can drag and drop the file anywhere on the window?

Reply:
@josxha I have updated i18n to make it clear that this is only for PMTiles